### PR TITLE
fix delete-db not working on a truly clean OS

### PR DESCRIPTION
**Pull request by Pomax**
Closes https://github.com/mozilla/foundation.mozilla.org/issues/3609

@patjouk I wasn't sure if there was a way to properly check for the volume so instead of an `if` I ended up using a `try` with a passing except... would you happen to know if this can be done cleaner?

### CODE REVIEW

File: tasks.py [hunk 1]
```diff
@@ -223,22 +215,31 @@ def docker_switching_branch(ctx):
...
-    print("Creating fake data")
-    docker_manage(ctx, "load_fake_data")
-    print("Updating localizable fields")
-    docker_l10n_sync(ctx)
-    docker_l10n_update(ctx)
-    print("Updating block information")
-    docker_manage(ctx, "block_inventory")
+    try:
+        ctx.run("docker volume rm foundationmozillaorg_postgres_data")
+    except:
```
Comment (Pomax) [inline tasks.py]: flake8 doesn't allow a bare except, but I don't know how to find out what the actual error type is: the log doesn't help in this case.

File: tasks.py [hunk 2]
```diff
@@ -223,22 +215,30 @@ def docker_switching_branch(ctx):
...
-    print("Creating fake data")
-    docker_manage(ctx, "load_fake_data")
-    print("Updating localizable fields")
-    docker_l10n_sync(ctx)
-    docker_l10n_update(ctx)
-    print("Updating block information")
-    docker_manage(ctx, "block_inventory")
+    try:
+        ctx.run("docker volume rm foundationmozillaorg_postgres_data")
+    except:  # noqa: E722
```
Comment (Pomax) [inline tasks.py]: I don't know how to get the "real" exception so that we're not doing a bare except - the log doesn't tell us in this case.

**Comment by Pomax** [comment]
@patjouk does this look sensible?

**Comment by patjouk** [comment]
I took a look and can't find a quick solution. Can you add a code comment on why you're adding this change and a print so that the user know it's an expected error?

**Comment by Pomax** [comment]
Good call - I've added both the code comment and print.

**Comment by Pomax** [comment]
@patjouk is this good, or were there other things to address too?

**Comment by patjouk** [comment]
There's one thing I would like to try tomorrow, if that's okay with you.

File: tasks.py (hunk 2 above)
Comment (cadecairos) [reply tasks.py]: Read through the invoke source, I think it raises one of these exceptions: https://github.com/pyinvoke/invoke/blob/aa5a02bfd7e6a73ceed11f0a94cf30d4a422b8aa/invoke/runners.py#L276-L286

**Comment by patjouk** [comment]
I think this can be closed because of #3673.

**Pull request closed without merging**